stacks-core/release/create-source-binary/action.yml

```diff
@@ -72,12 +81,12 @@ runs:
         file: ${{ github.action_path }}/build-scripts/${{ env.DOCKERFILE }}
         outputs: type=local,dest=./release/${{ inputs.arch }}
         build-args: |
-          STACKS_NODE_VERSION=${{ inputs.tag || env.GITHUB_SHA_SHORT }}
+          STACKS_NODE_VERSION=${{ inputs.node_tag || inputs.signer_docker_tag || env.GITHUB_SHA_SHORT }}
```

Comment:
Seems a bit odd to have `STACKS_NODE_VERSION` be set to inputs.node_tag _OR_ inputs.signer_docker_tag. I'm likely missing some context which may explain why this is needed, but just wanted to call it out in case it's an accident.

Also since node_tag and signer_docker_tag are required, I don't think the OR conditions will ever be hit.

Reply:
it is a bit messy - but we don't always build a stacks-core release, it may only be a signer release. 
and in the case of a stacks-core + signer release - we need to tag them slightly different. 

there may be a way to simplify this, but i'm not seeing it